### DISCUSSION TITLE
How to disable shift arrows?

### DISCUSSION THREAD

**Discussion by Crispy13** [mg979/vim-visual-multi#267] [Q&A]
I can't find it on wiki - keymap customize

After entering insert mode from multi mode, I would like pressing ESC to exit both insert mode and multi mode. Is there a way to do this?

For now I need to press ESC twice once for insert, once for multi-mode

**Comment by linyejoe2** [comment]
I have same issue.

Check out [docs](https://github.com/mg979/vim-visual-multi/wiki/Mappings#customization) and [source code](https://github.com/mg979/vim-visual-multi/blob/master/autoload/vm/maps/all.vim)

you can disable it by 
```bash
let g:VM_maps['Select l'] = '' 
let g:VM_maps['Select h'] = '' 
```

but at the end, I just disable default mappings through `let g:VM_default_mappings = 0`, because I don't need any of those key bind.
